### DISCUSSION TITLE
RESTeasy 6.0.1.FINAL doesnt work with latest Jackson (2.13.2)

### DISCUSSION THREAD

**Discussion by salokiam** [resteasy/resteasy#3097] [General]
When I try to compile resteasy-jackson2-provider Version 6.0.1.FINAL with newer version of Jackson, I get a compile error not finding  jackson-jaxrs-json-provider-2.13.2-jakarta.jar on maven central.

To reproduce you just need use this small compile script (its gradle):
`plugins {
    id 'java'
}

group 'org.example'
version '1.0-SNAPSHOT'

repositories {
    mavenCentral()
}

dependencies {
    implementation("org.jboss.resteasy:resteasy-jdk-http:6.0.1.Final")
    implementation("org.jboss.resteasy:resteasy-jackson2-provider:6.0.1.Final")
//    implementation ("com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.12.6")
    implementation ("com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.13.2")

//    implementation ("io.swagger.core.v3:swagger-jaxrs2-jakarta:2.2.0")
}
`

The fulle error message is:
`Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find jackson-jaxrs-base-2.13.2-jakarta.jar (com.fasterxml.jackson.jaxrs:jackson-jaxrs-base:2.13.2).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jaxrs/jackson-jaxrs-base/2.13.2/jackson-jaxrs-base-2.13.2-jakarta.jar
   > Could not find jackson-module-jaxb-annotations-2.13.2-jakarta.jar (com.fasterxml.jackson.module:jackson-module-jaxb-annotations:2.13.2).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/fasterxml/jackson/module/jackson-module-jaxb-annotations/2.13.2/jackson-module-jaxb-annotations-2.13.2-jakarta.jar
   > Could not find jackson-jaxrs-json-provider-2.13.2-jakarta.jar (com.fasterxml.jackson.jaxrs:jackson-jaxrs-json-provider:2.13.2).
     Searched in the following locations:
         https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jaxrs/jackson-jaxrs-json-provider/2.13.2/jackson-jaxrs-json-provider-2.13.2-jakarta.jar

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
`

I can provide the full project as well, if needed. (But will be still gradle :-)

**Comment by liweinan** [comment]
@salokiam In release note : https://github.com/resteasy/resteasy/releases/tag/6.0.1.Final

The supported Jackson version in this release is `2.12.6`:

<img width="912" alt="image" src="https://user-images.githubusercontent.com/201907/167439032-73aea8ab-5536-4628-b45c-f460871b0109.png">

Each RESTEasy release has it own supported Jackson version. If you do need to use Jackson `2.13.x`, better using the compatible RESTEasy version.

**Comment by jamezp** [comment]
[RESTEAST-3085](https://issues.redhat.com/browse/RESTEASY-3085) has been filed for the upgrade to 2.13. However, this will only be for 6.1.0.
